config.go

```diff
@@ -86,6 +86,10 @@ const (
 	// HostAnnouncer will wait between DNS resolutions to check if the
 	// backing IP of a host has changed.
 	defaultHostSampleInterval = time.Minute * 5
+
+	defaultChainInterval = time.Minute * 10
```

Comment:
Hmm, this seems a bit too high...I'd imagine we'd want to check at least once a minute, just so any hypervisor-like monitor can restart `lnd` shortly after the backend shuts down.

Reply:
An alternative to polling would be to rely on the `zmq` connection for `bitcoind` and the websockets connection for `btcd`. If either of these connections fail (EOF error), then we know the backend is down.

Reply:
Relying on these connections meaning make a change in `btcwallet` so that we can access the existing connection, or add a direct connection to zmq/websockets from the health check?

Reply:
Thinking just a direction connection so we don't need to dig down into the wallet. For `bitcoind`, we can only request the block notifications which aren't as spammy. A similar option can be applied to the `btcd` web sockets connection.